src/entities/uniqueAssets.ts

```diff
@@ -68,3 +62,63 @@ export interface UniqueAssetTrait {
   display_type: string;
   max_value: string | number | null | undefined;
 }
+
+export interface SeaportOrder {
```

Comment:
This was manually copied out but could there be a nicer way of doing this? Maybe a @types/ package?